_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 5 tips to keep your company Instagram channel alive / 5 tips on how to manage your company Instagram channel
+date: 2020-06-04
+header_image: public/instagram-company-account.jpg
+header_position: center
+header_overlay: true
+category: techcomm-and-socialmedia
+tags: ["social media", "Instagram", "content"]
+authors: ["Aroso"]
+about_authors: ["amahaiuddin"]
+---
+
+It is called a picture is worth a thousand words - Instagram is the perfect social media platform for sharing visuals around the world.
+We started our Instagram channel[@life_at_epages](https://www.instagram.com/life_at_epages/){:target="_blank"} in 2019 to present our company culture and express thus who we are.
+But managing a company Instagram channel and coordinating content involves some important aspects to consider.
+In the following I will explain which actions we take in order to keep our company Instagram channel alive.
+
+## Create Instagram How-to guide
+
+Usually, an Instagram company channel is not run by a single person but by a team consisting of two or more employees.
+In order to develop and create consistent content for our Instagram channel, we created an Instagram How-to guide.
+This guide shows the employees an overview of how to use Instagram at ePages and is continuously being updated.
+
+- **Aim of Instagram channel**:
+Write down what you want to reach with the Instagram channel and in which areas the channel should support the company.
+Should it support you more in promoting your company product, should it support you in reaching out to potential hires, or should it support you more in gaining traffic on your company blog.
```

Comment:
Quite some duplication. Would shorten it a bit: Should it e.g. promote your products, support you in reaching out to potential hires, or help you to gain more traffic on your company blog?